utils/StdExtensions.h

```diff
@@ -82,6 +82,12 @@ struct hash<char*>
     uint64_t operator()(const char* s, size_t len) const { return hashBuffer(s, len); }
 };
 
+template <>
+struct hash<const char*>
+{
+    uint64_t operator()(const char* s) const { return hash<char*>{}(const_cast<char*>(s)); }
```

Comment:
you could avoid the `const_cast` if you change the current `utils::hash<char*> to utils::hash<const char*>` and make this one the non-const specialisation

Reply:
🤔  Or perhaps you don't need the `const_cast` here as `utils::hash<char *>` receives a `const char*`